addons/l10n_es_edi_sii/models/mail_template.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    def _get_edi_attachments(self, document):
+        # When an invoice is signed, an electronic document is created (i.e. jsondump.json)
+        # Prevent this document to be added in "SEND & PRINT" wizard
+        if document.name == 'jsondump.json' and document.edi_format_id.code == 'es_sii':
+            return {}
```

Comment:
I'm actually not sure this would be the best course of action :thinking: I'd rather not add a whole file in stable to fix a bug. 
I think we need to go deeper. Why is the json file generated ? Where ? Why exclusively in l10n_es ? 

Did you have a chance to understand this behavior ?

Reply:
Thanks for the review.

I don't know if there is a reason to create this file, but it is created intentionally here: https://github.com/odoo/odoo/blob/0ae818cca87bd8433a0783797309771125772d86/addons/l10n_es_edi_sii/models/account_edi_format.py#L685-L692

Reply:
After discussing with JCO, it comes out that we are creating this file to allow customer to check the data that have been sent to the E-invoicing service.